server/src/test/java/org/opensearch/index/engine/NRTReplicationEngineTests.java

```diff
@@ -203,6 +203,36 @@ public void testSimultaneousEngineCloseAndCommit() throws IOException, Interrupt
         }
     }
 
+    public void testSimultaneousFlushAndCommit() throws IOException, InterruptedException {
```

Comment:
Don't see any assertions been made. Do you mind adding a comment around what functionality does this test validates.

Reply:
hm yea this is useless, i've removed it.